### PR TITLE
[IMP] search by email_from and reply_to

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Introduce search of html fields
Create indexes
Update Readme

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
@darshan-serpent please include those changes in your branch

File: base_search_mail_content/__openerp__.py [hunk 1]
```diff
@@ -5,14 +5,15 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
-    "name": "Mail message name search",
+    "name": "Base Mail Search Content",
```
Comment (serpentcs-dev1) [inline base_search_mail_content/__openerp__.py:8]: Base Search Mail Content

File: base_search_mail_content/README.rst [hunk 2]
```diff
@@ -3,43 +3,45 @@
...
 
 ========================
-Mail message name search
+Base Search Mail Content
 ========================
 
-This module adds the capability to search for mail messages by subject or
-body of the message. This will be useful in models that make intense use of
-messages, like project issues or helpdesk tickets.
+This module adds the capability to find any object (e.g. project issues or
```
Comment (serpentcs-dev1) [inline base_search_mail_content/README.rst:9]: Should be : This module adds the capability to find **on** any object (e.g. project issues or

**Review by serpentcs-dev1 — APPROVED** [review]

**Comment by serpentcs-dev1** [comment]
@aheficent Thanks, I'll do the necessary changes.

**Pull request merged**